packages/pcd/message-pcd/test/MessagePCD.spec.ts

```diff
@@ -71,7 +71,7 @@ describe("message-pcd should work", function () {
     /////////////////////////////////
     const copy3 = await MessagePCDPackage.deserialize(ser.pcd);
     expect(true).to.eq(await MessagePCDPackage.verify(copy3));
-    copy3.proof.signature.claim.publicKey[0] += "1";
+    copy3.proof.signature.claim.publicKey += "1";
```

Comment:
This should probably be a well-formed public key in order to truly be testing the same thing.  I'd expect this test to fail to deserialize, long before it gets to trying to verify a proof.